tests/test_util.py

```diff
@@ -45,3 +54,15 @@ def test_warn(capsys):
     util.warn('test warn')
     _, err = capsys.readouterr()
     assert '!!! WARNING: test warn' in err
+
+
+def test_import_app():
+    assert util.import_app("test:app")
+
+    with pytest.raises(ImportError) as e_info:
+        util.import_app("a:app")
+    assert "No module" in str(e_info)
+    
+    with pytest.raises(AppImportError) as e_info:
```

Comment:
Style nit: `err` is more common name in this file. I'd suggest using it to be consistent with the rest of the file.

Reply:
OK